packages/dataviews/src/types.ts

```diff
@@ -596,6 +597,29 @@ export interface ActionButton< Item > extends ActionBase< Item > {
 
 export type Action< Item > = ActionModal< Item > | ActionButton< Item >;
 
+export interface EmptyViewProps {
+	/**
+	 * The message to display when the view is empty.
+	 */
+	heading: string;
+
+	/**
+	 * The message to display when the view is empty.
+	 */
+	description?: string;
+
+	/**
+	 * The illustration to display when the view is empty.
+	 * It can be a string (URL) or a React element.
+	 */
+	illustration?: string | ReactElement;
```

Comment:
Accepting an arbitrary `ReactElement` could be abused. But it means that an `<Icon>` could be used for example.